packages/edition-twig/patternlab-config.default.js

```diff
@@ -0,0 +1,122 @@
+const resolvePkg = require('resolve-pkg');
+
+module.exports = {
+  engines: {
+    twig: {
+      namespaces: [
+        {
+          id: 'uikit',
+          recursive: true,
+          paths: [resolvePkg(`@pattern-lab/uikit-workshop/views-twig`)],
+        },
+        {
+          id: 'atoms',
+          recursive: true,
+          paths: ['source/_patterns/00-atoms'],
+        },
+        {
+          id: 'molecules',
+          recursive: true,
+          paths: ['source/_patterns/01-molecules'],
+        },
+      ],
+      alterTwigEnv: [
+        {
+          file: 'alter-twig.php',
+          functions: ['addCustomExtension'],
+        },
+      ],
+    },
+  },
+  cacheBust: true,
+  cleanPublic: true,
+  defaultPattern: 'all',
+  defaultShowPatternInfo: false,
+  ishControlsHide: {
+    s: false,
+    m: false,
+    l: false,
+    full: false,
+    random: false,
+    disco: false,
+    hay: true,
+    mqs: false,
+    find: false,
+    'views-all': false,
+    'views-annotations': false,
+    'views-code': false,
+    'views-new': false,
+    'tools-all': false,
+    'tools-docs': false,
+  },
+  ishViewportRange: {
+    s: [240, 500],
+    m: [500, 800],
+    l: [800, 2600],
+  },
+  logLevel: 'info',
+  outputFileSuffixes: {
+    rendered: '.rendered',
+    rawTemplate: '',
+    markupOnly: '.markup-only',
+  },
+  paths: {
+    source: {
+      root: './source/',
+      patterns: './source/_patterns/',
+      data: './source/_data/',
+      meta: './source/_meta/',
+      annotations: './source/_annotations/',
+      styleguide: 'dist/',
+      patternlabFiles: {
+        'general-header':
+          '../uikit-workshop/views/partials/general-header.mustache',
+        'general-footer':
+          '../uikit-workshop/views/partials/general-footer.mustache',
+        patternSection:
+          '../uikit-workshop/views/partials/patternSection.mustache',
+        patternSectionSubtype:
+          '../uikit-workshop/views/partials/patternSectionSubtype.mustache',
+        viewall: '../uikit-workshop/views/viewall.mustache',
```

Comment:
Wouldn't we need `require.resolve()` on all these `uikit-workshop` paths too?

Reply:
See #1225